operator/src/main/resources/scripts/introspectDomain.sh

```diff
@@ -74,6 +87,17 @@ for dir_var in DOMAIN_HOME JAVA_HOME WL_HOME MW_HOME ORACLE_HOME; do
   [ ! -d "${!dir_var}" ] && trace SEVERE "Missing ${dir_var} directory '${!dir_var}'." && exit 1
 done
 
+trace "DATA_HOME=${DATA_HOME}"
+
+#
+# DATA_HOME env variable exists implies override directory specified.  Attempt to create directory
+#
+if [ ! -z "${DATA_HOME}" ] && [ ! -d "${DATA_HOME}" ]; then
+  createFolder ${DATA_HOME}
+  trace "Created data home directory: '${DATA_HOME}'"
```

Comment:
Move the trace statement before 'createFolder' so that if createFolder fails the tracing will show why it was being called.   Change 'Created' to 'Creating'.